ranger/gui/ui.py

```diff
@@ -182,11 +185,6 @@ def destroy(self):
                                   'automatic-rename', self._tmux_automatic_rename])
                 except CalledProcessError:
                     pass
-            else:
```

Comment:
I don't understand why you'd remove restoring the option to unset? Is the option unset automatically when we issue `rename-window`?

Reply:
As far as I understand it is. When I start tmux `automatic-rename` is `on` by default

When I issue a `rename-window` it turns to `off`